ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+```
```

Comment:
Instead of making a code block, maybe wrap this in `<!--` and `-->` which will comment it out when the issue is created.